internal/server/iigointernal/orchestration.go

```diff
@@ -6,13 +6,16 @@ import (
 	"github.com/SOMAS2020/SOMAS2020/internal/common/roles"
 	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
 	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/voting"
 	"github.com/pkg/errors"
 )
 
+const serviceCharge = shared.Resources(10)
+
 // featureJudge is an instantiation of the Judge interface
 // with both the Base Judge features and a reference to client judges
-var featureJudge = BaseJudge{
-	ID:                0,
+var judicialBranch = judiciary{
```

Comment:
https://www.safetyresearch.net/blog/articles/toyota-unintended-acceleration-and-big-bowl-%E2%80%9Cspaghetti%E2%80%9D-code#:~:text=Toyota%20had%20more%20than%2010%2C000,%2C%20ten%2C%20okay%2C%20fine.&text=He%20was%20critical%20of%20Toyota,death%20of%20a%20major%20task.

Reply:
#125

Reply:
Ok I've implemented a compromise here. These branches are still globals purely for the sake of persistence over IIGO runs, any functions apart from RunIIGO which access these are fed them in their function signature